common-libs/common/src/Affinity.ts

```diff
@@ -25,10 +25,14 @@ export class Affinity {
   private readonly _platformCryptographyTools
 
   constructor(options: AffinityOptions, platformCryptographyTools: IPlatformCryptographyTools) {
-    this._didResolver = new DidResolver({
-      registryUrl: options.registryUrl ?? resolveUrl(Service.REGISTRY, 'staging'),
-      accessApiKey: options.apiKey,
-    })
+    this._didResolver =
+      options.didResolver ??
+      new LocalDidResolver({
+        registryUrl: options.registryUrl ?? resolveUrl(Service.REGISTRY, 'staging'),
```

Comment:
Why hard coded staging? This seems to have been a previous oversight :|

Reply:
It's not the only place in Affinidi.ts where resolveUrl(Service.REGISTRY, 'staging') is used. I don't know why it's hardcoded and if it should be replaced.